packages/react-components/react-progress/src/components/Progress/Progress.types.ts

```diff
@@ -2,45 +2,31 @@ import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utili
 
 export type ProgressSlots = {
   /**
-   * The root of the Progress
+   * The root of the Progress. This slot will also function as the Progress's track
    * The root slot receives the `className` and `style` specified directly on the `<Progress>`.
    */
   root: NonNullable<Slot<'div'>>;
-  /**
-   * The title of the Progress.
-   * The label slot receives the styling related to the title associated with the Progress.
-   */
-  label?: Slot<'span'>;
   /**
    * The animated slot of the Progress
    * The bar slot receives the styling related to the loading bar associated with the Progress
    */
   bar?: NonNullable<Slot<'div'>>;
-  /**
-   * The track slot of the Progress
-   * The track slot receives the styling related to the loading bar track associated with the Progress
-   */
-  track?: NonNullable<Slot<'div'>>;
-  /**
-   * The description slot of the Progress
-   * The description slot receives the styling related to the description associated with the Progress
-   */
-  description?: Slot<'span'>;
 };
 
 /**
  * Progress Props
  */
 export type ProgressProps = Omit<ComponentProps<ProgressSlots>, 'size'> & {
   /**
-   * Prop to set whether the Progress is determinate or indeterminate
-   * @default false
+   * Percentage of the operation's completeness of the determinate progress
+   * in decimal format, numerically between 0 and 1.
```

Comment:
It's not really a percentage. How about this?
```suggestion
   * A decimal number between `0` and `1` (or between `0` and `max` if given),
   * which specifies how much of the task has been completed. 
   *
   * If `undefined` (default), the Progress will display an **indeterminate** state.
```